modules/openapi/tests/src/ExtensionsSpec.scala

```diff
@@ -151,4 +151,76 @@ final class ExtensionsSpec extends munit.FunSuite {
     TestUtils.runConversionTest(openapiString, expectedString)
   }
 
+  test("extensions: doesn't capture x-format of alloy time types".only) {
```

Comment:
remove the `.only` test option
```suggestion
  test("extensions: doesn't capture x-format of alloy time types") {
```